src/foremast/runner.py

```diff
@@ -130,11 +132,19 @@ def create_pipeline(self, onetime=None):
 
         spinnakerpipeline.create_pipeline()
 
-    def create_iam(self):
-        """Create IAM resources."""
-        utils.banner("Creating IAM")
+    def create_aws_iam(self):
+        """Create AWS IAM resources."""
+        utils.banner("Creating AWS IAM")
         iam.create_iam_resources(env=self.env, app=self.app)
 
+    def create_gcp_iam(self, env: GcpEnvironment):
```

Comment:
Assuming split was made with some thought. No ability under iam to have gcp and aws. if not in this release, ability to refactor in the future? Not sure if much code similarities or not between AWS/GCP.

Reply:
The implementations are very different for sure, but if there were a pipeline type that needed both it wouldn't be hard to invoke both sides of Foremast.  There would be a bit more to think about like, do environments in AWS match the names in GCP and things like that but definitely doable